package-scripts.js

```diff
@@ -17,6 +17,10 @@ module.exports = {
       watch: "karma start --auto-watch ./config/karma/karma.conf.js",
       default: "karma start ./config/karma/karma.conf.js"
     },
+    jest: {
+      native: "jest --config=jest-native-config.js",
+      default: "jest --config=jest-native-config.js"
+    },
```

Comment:
I'm not 100% sure what this should look like, so I left it like this until we visit the Lerna setup more generally.